docworks-cli/test/tern-e2e.spec.js

```diff
@@ -50,6 +50,36 @@ describe('tern workflow e2e', function () {
     )
   })
 
+  it('generate tern from a remote repo\'s non-master branch', async function () {
```

Comment:
maybe it's just me or the hour 🙂 but I got a bit confuse about this test and the previous one(maybe because they return the same value).
What do you think about returning a different value(from the `test-utils`) in case of a branch? for example, `test/ver3` contains also `newOperation` property.

Reply:
it's a good idea. maybe to even just create a repo with each version in its own branch for all tests.
but i think i'll leave it for next time. i don't want to invest too much here since i hope it'll get deprecated pretty soon.